### PR TITLE
ZCS-901: Universal UI: Modernize the style of the actions menu tag dropdown

### PR DESCRIPTION
**Pull request by tarangkhandelwal**
ZCS-901: Universal UI: Modernize the style of the actions menu tag dropdown

Changeset:

1. ZmMsg.properties:
	* Removing obsolete property removeTag.
	* Updating allTags property value.
2. ZmTagMenu.js:
	* Simplifying the tags menu: All tags in single list and showing added tags with a checkmark.
	* Removing code for remove tag menu.
	* Removing menu item remove tag ( when only 1 tag added).

### CODE REVIEW

File: WebRoot/js/zimbraMail/share/view/ZmTagMenu.js [hunk 1]
```diff
@@ -183,8 +183,27 @@ function(tagList, addRemove) {
 		var tagName = addRemove.add[i];
 		this._addNewTag(this, tagName, tagList, true, null, this._addHash);
 	}
+	
+	var removeList = addRemove.remove;
+	if (removeList.length > 0) {
+			for (i = 0; i < removeList.length; i++) {
```
Comment (silentsakky) [inline WebRoot/js/zimbraMail/share/view/ZmTagMenu.js]: indentation has one more tab for entire code

Comment (tarangkhandelwal) [inline WebRoot/js/zimbraMail/share/view/ZmTagMenu.js]: Fixed. Please check now

File: WebRoot/js/zimbraMail/share/view/ZmTagMenu.js [hunk 2]
```diff
@@ -183,8 +183,27 @@ function(tagList, addRemove) {
...
+	var removeList = addRemove.remove;
+	if (removeList.length > 0) {
+		for (i = 0; i < removeList.length; i++) {
+			var tagName = removeList[i];
+			var tagHtmlId = 'Remove_tag_' + i;
+			this._addNewTag(this, tagName, tagList, false, null, this._removeHash, tagHtmlId);
+		}
+		//remove all tags
+		new DwtMenuItem({parent:this, style:DwtMenuItem.SEPARATOR_STYLE});
+		var mi = new DwtMenuItem({parent:this, id:"REMOVE_ALL_TAGS", className: "ZAddNewMenuItem"});
```
Comment (silentsakky) [inline WebRoot/js/zimbraMail/share/view/ZmTagMenu.js]: should we have some different class name? ZAddNewMenuItem indicates addition of tags

Comment (tarangkhandelwal) [inline WebRoot/js/zimbraMail/share/view/ZmTagMenu.js]: its a generic class name I guess Kalpesh added before which I am just using it to style the add items DwtMenuItem differently.

**Pull request merged**